### PR TITLE
Perception: add ultrasonic subnode framework

### PR DESCRIPTION
**Pull request by kechxu**

### CODE REVIEW

File: modules/perception/proto/perception_ultrasonic.proto [hunk 1]
```diff
@@ -0,0 +1,18 @@
+syntax = "proto2";
+
+package apollo.perception;
+
+import "modules/common/proto/header.proto";
+import "modules/common/proto/geometry.proto";
+
+message ImpendingCollisionEdge {
+    optional int32 id = 1;  // edge ID.
+    repeated int32 cone_id_list = 2;  // cone ID.
```
Comment (Capri2014) [inline modules/perception/proto/perception_ultrasonic.proto]: cone ID?

**Comment by Capri2014** [comment]
@kechxu How would you plan to add sonars calibration parameters? load from configs in perception? or you want can bus to publish both range/calibration datas? @kechxu @techoe @gchen-apollo @gengqx @weidezhang

File: modules/perception/proto/perception_ultrasonic.proto (hunk 1 above)
Comment (kechxu) [reply modules/perception/proto/perception_ultrasonic.proto]: Yes

**Comment by techoe** [comment]
Awesome! Thank you so much!

**Pull request merged**